docs/tools/terraform/reference/cookbook/multicloud-postgresql-recipe.rst

```diff
@@ -0,0 +1,71 @@
+Deploy PostgreSQL® services to multiple clouds and regions
+==========================================================
+
+There are many reasons why businesses need to distribute their databases geographically. One of the primary reason is data residency/regulation which mandates user data to stay within certain regions. 
+This example shows how to set up three highly-available PostgreSQL databases in three regions and across different cloud providers for data regulation and compliance. In addition, these databases must be configured in a way that they are protected against database deletion.
+`Aiven Terraform Provider <https://registry.terraform.io/providers/aiven/aiven/latest/docs>`_ can help create multiple services programmatically. 
+
+The following image shows that the Aiven Terraform Provider calls the Aiven API under the hood to create three PostgreSQL services on AWS (Europe), DigitalOcean (US), and Google Cloud Platform(Asia):
+
+.. mermaid::
+
+   graph LR
+      A[Aiven Terraform Provider]
```

Comment:
I don't think the provider should be in the picture, the recipe spins up three standalone databases. Let's also name the elements "PostgreSQL (cloud name)" so that we can be somewhat consistent in showing what Aiven things each script creates.

Reply:
I wasn't sure what to replace the provider with so I renamed the block as *Terraform Script*. I also used the full name i.e. *Aiven for PostgreSQL - AWS EU*. Please let me know if this is what you meant. I can make another quick edit if in case I didn't understand the change request.

Reply:
I am so confused, this comment wasn't visible when I reviewed this PR 2 minutes ago. Anyway, I did the thing I thought the diagram needed.